### ISSUE TITLE
Cannot Find 'Subnet' Shape

### ISSUE THREAD

**Issue by brianstringfellow** [sandroasp/Microsoft-Integration-and-Azure-Stencils-Pack-for-Visio#47]
I do not see the Subnet shape in the stencil pack. I also cannot find it in the SVG collection in the repository.

From the Azure Portal:
![image](https://user-images.githubusercontent.com/22823524/159781756-d35acfb2-0cbb-46c3-ae5e-abf03749bf97.png)

**Comment by sandroasp** [comment]
The Subnet shape is available on the Microsoft Azure: Additional or Support
![image](https://user-images.githubusercontent.com/1318263/160496103-6c4b8aa6-310b-4645-a723-ed64d62b1123.png)

**Comment by brianstringfellow** [comment]
In case anyone else takes the same path I originally took, I had followed the manual installation process and downloaded ONLY the [MIS Azure Stencils.vssx](https://github.com/sandroasp/Microsoft-Integration-and-Azure-Stencils-Pack-for-Visio/blob/master/Azure/MIS%20Azure%20Stencils.vssx) file and dropped that into my **My Shapes** folder. That did not provide all the stencil files that I needed.

I went back and downloaded [download-Install-MIS-Azure-Visio-Package.ps1](https://github.com/sandroasp/Microsoft-Integration-and-Azure-Stencils-Pack-for-Visio/blob/master/download-Install-MIS-Azure-Visio-Package.ps1), then executed that file. Now I have all the stencils in place and can find the shapes I need.
